spectral_cube/io/fits.py

```diff
@@ -135,12 +141,13 @@ def load_fits_cube(input, hdu=0, meta=None, **kwargs):
 
         data, wcs = cube_utils._split_stokes(data, wcs)
 
-        mask = {}
+        stokes_data = {}
         for component in data:
-            data[component], wcs_slice = cube_utils._orient(data[component], wcs)
-            mask[component] = LazyMask(np.isfinite, data=data[component], wcs=wcs_slice)
+            comp_data, comp_wcs = cube_utils._orient(data[component], wcs)
+            comp_mask = LazyMask(np.isfinite, data=comp_data, wcs=comp_wcs)
+            stokes_data[component] = SpectralCube(comp_data, wcs=comp_wcs, mask=comp_mask, meta=meta, header=header)
```

Comment:
Wrap this at 80 chars?  (and related PEP8 if there are other things, but I assume you have done those right everywhere without checking)